src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+export const Person = ({
```

Comment:
https://github.com/mate-academy/react_person/pull/1874#discussion_r1402045204 - not fixed 😢